- [x] 有个单测报错：`tests/distributed/test_chunked_moe.py`
ERROR: test_case (__main__.TestChunkedMoE)
```shell
   Traceback (most recent call last):
     File "/workspace/FastDeploy/tests/distributed/chunked_moe.py", line 185, in test_case
       self.run_model_runner()
     File "/workspace/FastDeploy/tests/distributed/chunked_moe.py", line 156, in run_model_runner
       self.model_runner.initialize_forward_meta()
     File "/workspace/FastDeploy/fastdeploy/worker/gpu_model_runner.py", line 1593, in initialize_forward_meta
       if self.graph_opt_config.graph_opt_level > 0 and not self.graph_opt_config.full_cuda_graph:
   AttributeError: 'GPUModelRunner' object has no attribute 'graph_opt_config'
```

--- a/fastdeploy/config.py
+++ b/fastdeploy/config.py
@@ -924,6 +924,10 @@ class GraphOptimizationConfig:
         self.use_unique_memory_pool: bool = True
         """ Whether to use cudagraph for draft model."""
         self.draft_model_use_cudagraph: bool = False
+        """ Maximum CUDA Graph capture size for static graph mode.
+        Recommend 512 for small models (e.g., ERNIE45T 0.3B) and 128 for massive models (e.g., 300B).
+        """
+        self.max_capture_shape_dy2st: int = 512
 
         # CINN Config ...
         if args is not None:
@@ -1650,8 +1654,7 @@ class FDConfig:
             max_capture_shape = min(512, max_capture_shape)
 
         if self.graph_opt_config.graph_opt_level > 0:
-            # TODO(DrRyanHuang): Tune this value or expose it as a CLI argument.
-            max_capture_shape = 512
+            max_capture_shape = graph_opt_config.max_capture_shape_dy2st
 
         if self.graph_opt_config.cudagraph_capture_sizes is None:
             dec_token_per_query_per_step = (

--- a/fastdeploy/worker/gpu_model_runner.py
+++ b/fastdeploy/worker/gpu_model_runner.py
@@ -1590,7 +1590,11 @@ class GPUModelRunner(ModelRunnerBase):
         )
 
         # Use static graph splitting to isolate incompatible operators from the CUDA Graph. This splits the graph into subgraphs, allowing Prefill, Decode, and Mixed Batches to run compatible parts via CUDA Graph.
-        if self.graph_opt_config.graph_opt_level > 0 and not self.graph_opt_config.full_cuda_graph:
+        if (
+            hasattr(self, "graph_opt_config")
+            and self.graph_opt_config.graph_opt_level > 0
+            and not self.graph_opt_config.full_cuda_graph
+        ):
             self.forward_meta.step_use_cudagraph = True
 
         # Set forward_meta.is_dummy_or_profile_run to True to skip init_kv_signal_per_query for attention backends

--- a/fastdeploy/worker/gpu_model_runner.py
+++ b/fastdeploy/worker/gpu_model_runner.py
@@ -1592,6 +1592,7 @@ class GPUModelRunner(ModelRunnerBase):
         # Use static graph splitting to isolate incompatible operators from the CUDA Graph. This splits the graph into subgraphs, allowing Prefill, Decode, and Mixed Batches to run compatible parts via CUDA Graph.
         if (
             hasattr(self, "graph_opt_config")
+            and self.use_cudagraph
             and self.graph_opt_config.graph_opt_level > 0
             and not self.graph_opt_config.full_cuda_graph
         ):

--- a/tests/e2e/test_EB_VL_Lite_sot_serving.py
+++ b/tests/e2e/test_EB_VL_Lite_sot_serving.py
@@ -91,7 +91,7 @@ def setup_and_run_server():
         "--reasoning-parser",
         "ernie-45-vl",
         "--graph-optimization-config",
-        '{"graph_opt_level": 2, "use_cudagraph": true, "full_cuda_graph": false}',
+        '{"graph_opt_level": 2, "use_cudagraph": true, "full_cuda_graph": true}',  # TODO(DrRyanHuang): we will support full_cuda_graph=false for VL model in next PR
     ]
 
     # Start subprocess in new process group

